tests/bottom_bar/__snapshots__/bottom_bar_component.test.ts.snap

```diff
@@ -168,7 +168,7 @@ exports[`BottomBar component simple rendering 1`] = `
   </div>
   
   <div
-    class="o-ripple-container position-relative"
+    class="o-ripple-container position-relative listSheets"
```

Comment:
missing bottom bar reactivity tests

Reply:
I don't really see what I can test except a snapshot tbh but at any given change, we'll just update it blindly. Vincent suggested that the tests could be added in odoo to have an actual browser and not mocks everywhere.

Reply:
You only have to mock `env.isSmall` to test what happens to the bottom bar no ?